README.md

```diff
@@ -256,7 +256,7 @@ ansible_connection: ansible.netcommon.httpapi
 ansible_httpapi_port: 443
 ansible_httpapi_use_ssl: true
 ansible_httpapi_validate_certs: false
-ansible_network_os: cisco.dcnm.dcnm
+ansible_network_os: cisco.nd.nd
```

Comment:
I had a discussion with @mtarking and we agreed that it will be better if we can dynamically pick which module we call `cisco.dcnm.dcnm_rest` vs `cisco.nd.nd_rest` based on the `ansible_network_os` setting.  That way we can have a control knob.